devel-tools/refresh-ct-ca-lists.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
```

Comment:
Perhaps consider doing `set -euo pipefail`, as this is a new script :)

This does require a `set +e` before the `SED_VERSION` line and an `set -e` after, but it still feels nice to me to do that, especially if there is a chance that this will be used in an automated pipeline in the future.